engine/src/main/java/org/entando/entando/web/label/LabelValidator.java

```diff
@@ -50,4 +54,11 @@ public void validateBodyName(String labelCode, LabelRequest labelRequest, Errors
             errors.rejectValue("key", ERRCODE_URINAME_MISMATCH, new String[]{labelCode, labelRequest.getKey()}, "labelRequest.key.mismatch");
         }
     }
+
+    @Override
+    public boolean isValidField(String fieldName) {
```

Comment:
This should be done generically. Given that the reference is static it can be a parameter to the call.